### PR TITLE
Autodoc fix + Lava-DL landing page

### PR DESCRIPTION
**Pull request by bamsumit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,9 @@ pip install -r requirements.txt
 ```bash
```
Comment (mgkwill) [inline README.md:9]: Can we change this to "Test Doc Build"

**Review by mgkwill — CHANGES_REQUESTED** [review]
This looks pretty good to me.

Please add `project.plugin_depends_on("nbsphinx")` to build.py: 34 and address the redundancy of "generation" in the readme.

For the CI build I probably need to add pull of lava and lib code to ensure it builds correctly.

File: README.md [hunk 2]
```diff
@@ -9,3 +9,9 @@ pip install -r requirements.txt
...
 pyb sphinx_generate_documentation
```
Comment (mgkwill) [inline README.md:10]: Can we change this to "Test Doc Build"

**Comment by bamsumit** [comment]
@mgkwill: addressed your comments. Also added scripts to sync notebooks and render notebooks.

**Review by awintel — APPROVED** [review]
Thanks

**Review by mgkwill — APPROVED** [review]

**Pull request merged**